### DISCUSSION TITLE
Spancat Visualization with displacy

### DISCUSSION THREAD

**Discussion by adidonatocda** [explosion/spaCy#10249] [Help: Coding & Implementations] [answered]
I was able to build a pipeline with spancat. When I attempted to "evaluate" the model using the CLI I did not get any HTML output. I am assuming this is because the spancat feature is experimental and displacy only supports "ent" and "dep" styles 
(e.g. html = displacy.render(doc, style="dep")? 

Any direction or guidance on how I can get spancat visualizations similar to "ent" with displacy or other tool would be greatly appreciated.

This is the CLI command that I ran to attempt to get some html output.

```sh
python -m spacy evaluate  "E:\projectdir\spancatout\model-best" "E:\projectdir\spancat\dev.spacy" --output spancat-eval.json --displacy-path spancat-eval-html
```

**Comment by ljvmiranda921** [comment, marked as answer]
Hi @adidonatocda ,

You're correct, as it's experimental, Spans aren't supported in displacy. However, if you still wish to render them, a hacky way might be to copy the spans from `doc.spans[spans_key]` to ` doc.ents` while being careful of the overlaps, and render each entity one-by-one. This assumes that there are no overlaps within a span type. Once you've rendered each span type, you can combine them manually into a single HTML output. 

However, if you have overlapping spans within the same span type, this hacky way will not work.
